### ISSUE TITLE
Usage questions

### ISSUE THREAD

**Issue by ccamacho** [MaTeMaTuK/gantt-task-react#133]
This is a great Gantt chart! Thanks for the hard work, but I have some usage questions.

Is it possible to resize the Gantt chart to an specific width?
Is it possible to remove the From and To columns?
Im using the 0.3.8 version

Thanks!

**Comment by MatthewHazeTrinh** [comment]
**If you want to remove the From and To columns, you can do the following:**

You can copy paste this component: https://github.com/MaTeMaTuK/gantt-task-react/blob/main/src/components/task-list/task-list-header.tsx

And this: https://github.com/MaTeMaTuK/gantt-task-react/blob/main/src/components/task-list/task-list-table.tsx

Comment out or remove the divs as shown below:
![image](https://user-images.githubusercontent.com/117746862/201871074-f0d1d75a-d2f8-4ec8-80da-5b4983689317.png)
![image](https://user-images.githubusercontent.com/117746862/201871632-ce43664a-a3a1-4bf1-844e-e6548e8fd778.png)

And then add them to the Gantt component after importing them:
![image](https://user-images.githubusercontent.com/117746862/201871799-ef4f4cea-2698-4e40-804d-40d9f8ca9846.png)

This will overwrite the tasklistheader and tasklisttable with your own components.
This way, you can also add your own columns, which I did with a "Phase" column:
![image](https://user-images.githubusercontent.com/117746862/201872303-99cf0995-4226-4918-a6a7-dd8b43e3cdee.png)


**Regarding custom width, I don't know if it will solve your use-case, but I wrapped the Gantt task react component around a div, and specified a width, which looks like this:**
![image](https://user-images.githubusercontent.com/117746862/201872760-1d16e979-067e-4e0b-b5f1-323d04e648ba.png)
![image](https://user-images.githubusercontent.com/117746862/201872797-8b34d376-2b7e-49b5-b1e6-420d7b10b6a4.png)

**Comment by thegliche** [comment]
Hello Can one have multiple tasks on one row?

**Comment by elietech** [comment]
Yes, this would be most helpfull...multiple tasks on the same row...can we do this?
